src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1278,6 +1278,12 @@ class UpdateNodes : public UpdateNodesBase {
         m_completion.store(false);
         auto startCounter = m_prepareCounter.load();
 
+        // Allow nested parallel execution.
+        auto origin_nested_levels = get_max_nested_levels();
```

Comment:
Any details on that change? Why we need it?

Reply:
Some nodes use parallelism inside fn `updateDynParams`, but OMP has one nested level here at the moment, so nested routines can be executed on single thread only. This flag has higher priority than other like `num_threads`